### PR TITLE
add newfold module loader and coming soon module

### PR DESCRIPTION
**Pull request by circlecube**
Coming soon module.

### CODE REVIEW

**Comment by circlecube** [comment]
Would also love a code review on the module: https://github.com/newfold-labs/wp-module-coming-soon

File: bootstrap.php [hunk 1]
```diff
@@ -19,6 +22,64 @@
...
+);
+// Set coming soon values
+$nfd_module_container->set(
+	'comingsoon',
+	array(
+		'option_name'         => 'mm_coming_soon',
+		'admin_screen_id'     => 'web',
+		'admin_app_url'       => admin_url( 'admin.php?page=web#/home' ),
+		'template_h1'         => 'Coming Soon!',
+		'template_h2'         => 'A New WordPress Site',
```
Comment (wpscholar) [inline bootstrap.php:52]: @circlecube We should add translation functions for these two strings.

File: bootstrap.php [hunk 2]
```diff
@@ -19,6 +22,64 @@
...
+				)
+			);
+		}
+	)
+);
+// Set coming soon values
+$nfd_module_container->set(
+	'comingsoon',
+	array(
+		'option_name'         => 'mm_coming_soon',
```
Comment (wpscholar) [inline bootstrap.php:48]: We should probably just have an upgrade routine to update this option.

**Review by wpscholar — CHANGES_REQUESTED** [review]

File: bootstrap.php [hunk 3]
```diff
@@ -19,6 +22,64 @@
...
+			__( '%s &mdash; Coming Soon', 'wp-plugin-web' ),
+			esc_html( get_option( 'blogname' ) )
+		),
+		'admin_bar_text'       => '<div style="background-color: #FEC101; color: #000; padding: 0 1rem;">' . __( 'Coming Soon Active', 'wp-plugin-web' ) . '</div>',
+		'admin_notice_text'   => sprintf(
+			/* translators: %1$s is replaced with the opening link tag and %2$s is replaced with the closing link tag, %3$s is the opening link tag to preview the page, %4$s is the closing link tag. */
+			__( 'Your site is currently displaying a %3$sComing Soon page%4$s. Once you are ready, %1$slaunch your site%2$s.', 'wp-plugin-web' ),
+			'<a href="' . esc_url( admin_url( 'admin.php?page=web#/home' ) ) . '">',
+			'</a>',
+			'<a href="' . get_home_url() . '?preview=coming_soon" title="Preview the Coming Soon landing page">',
```
[the remark this answers is not in the document]
Comment (circlecube) [reply bootstrap.php:74]: Got it, Thanks!

File: bootstrap.php (hunk 1 above)
Comment (circlecube) [reply bootstrap.php:52]: Yes! Thanks

File: bootstrap.php (hunk 2 above)
Comment (circlecube) [reply bootstrap.php:48]: Now have an upgrade routine for this in place and migrating to use `nfd_coming_soon` in module and plugin.

File: bootstrap.php [hunk 4]
```diff
@@ -19,6 +22,64 @@
...
+			);
+		}
+	)
+);
+// Set coming soon values
+$nfd_module_container->set(
+	'comingsoon',
+	array(
+		'option_name'         => 'mm_coming_soon',
+		'admin_screen_id'     => 'web',
```
[the remark this answers is not in the document]
Comment (circlecube) [reply bootstrap.php:49]: Yes, good idea. Done

**Pull request merged**